CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
+## Maintainers' notice
+
+This file serves as a template for release notes in GitHub releases.
+When adding entries, please treat them as if they could end up in a release any time. This makes it much easier for us to make frequent releases!
+
+Thank you!
+
+# 0.18.19 - binary-breaking changes in `core`
+
+**WARNING**: This release includes binary-breaking changes in the `core` module. This is indirectly caused by an upstream change in [smithy-lang/smithy](https://github.com/smithy-lang/smithy/),
+and it'll most likely cause runtime issues in all applications that use a mix of versions in the ranges `0.18.0 ≤ 0.18.17` and `0.18.18+`.
```

Comment:
The **ONLY** way we can justify rolling the bump in 0.18 is by showing some confidence that it's VERY unlikely to cause runtime issues, unless people have written some very esoteric interpreters. 

That's what your experiments have shown (in our DMs), with the only way you've made it blow has been with this

```scala
SomeType.hints.get(Trait).map(_.breakingChanges.map(_.map(_.severity))))
```

So you need to change the wording to something : 

``` 
In the vast majority of applications using Smithy4s, it will not cause runtime issues. However, in the unlikely event that you have custom interpreters that query the `.breakingChanges` field of a `Trait` hint, you'll have to ensure that all the libraries pulled by your application are compiled against smithy4s 0.18.19. 
```

Reply:
I'll do some more tests to increase my confidence ;)

Reply:
updated the entry, does it look good?